scripts/data/cluster.sh

```diff
@@ -2,19 +2,19 @@
 
 #
 # Usage:
-# ./scripts/create_cluster_entity.sh
+# ./scripts/data/cluster.sh
 #
 # To use a different context
-# CONTEXT_TO_USE=my-ctx ./scripts/create_cluster_entity.sh
+# CONTEXT_TO_USE=my-ctx ./scripts/data/cluster.sh
 #
 # To create the cluster record on a Primaza server which is not localhost:8080
```

Comment:
I would say: to create the records on a different Primaza server

Reply:
Fixed with a70f38c